src/game/PowerGrid.hxx

```diff
@@ -0,0 +1,72 @@
+#ifndef POWERGRID_HXX_
+#define POWERGRID_HXX_
+
+#include "point.hxx"
+#include <vector>
+
+struct PowerNode
+{
+  Point coordinate;
+  int powerProduction = 0;
+};
+
+class PowerGrid;
+void mergePowerGrids(PowerGrid &mainGrid, PowerGrid &toBeMerged);
+class PowerGrid
+{
+public:
+  PowerGrid(PowerNode powerNode);
+
+  size_t size() { return m_powerNodes.size(); };
+
+  /**
+   * @brief Add a powernode to this powergrid
+   * 
+   * @param zoneNode to add
+   */
+  void addPowerNode(PowerNode powerNode);
+
+  /**
+   * @brief Remove a powernode on a given coordinate from this powerGrid
+   * 
+   * @param Coordinates of the mapNode with a power tile
```

Comment:
make sure to add the name of the parameter before the description, so in total "@param coordinate Coordinates of the mapNode with a power tile"